frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.jsx

```diff
@@ -87,12 +88,13 @@ const propertyLogItemTitle = (logItem) => {
     if (logItem.event_class_name === "BudgetLineItem") {
         title = `Budget Line ${propertyLabel} Edited`;
     } else if (logItem.event_class_name === "BudgetLineItemChangeRequest") {
+        title = logItem.property_key === "status" ? `Status Change to ${renderField(null, "status", logItem.change.new)}` : `Budget Change to ${propertyLabel}`;
         if (logItem.event_type === "IN_REVIEW") {
```

Comment:
IN_REVIEW, APPROVED, and REJECTED/DECLINED may be a good use case for an ENUM

Reply:
example of a JS fake ENUM:
```js
const BLI_STATUS = {
    DRAFT: "DRAFT",
    PLANNED: "PLANNED",
    EXECUTING: "IN_EXECUTION",
    OBLIGATED: "OBLIGATED"
};
```

Reply:
done